Lib/test/test_clinic.py

```diff
@@ -3294,7 +3294,15 @@ def test_meth_method_no_params(self):
         check = partial(self.assertRaisesRegex, TypeError, "no arguments")
         check(meth, 1)
         check(meth, a=1)
-        obj.test_meth_method_no_params_empty_kw_tuple()
+
+        from _testcapi import pyobject_vectorcall
```

Comment:
Since `_testcapi` is optional, the following tests can be skipped. I suggest to move them in a separate test method.

Reply:
Good call

Reply:
Addressed in eda7f03bb7a224b0b1fca821a8a88be90fa5cbc5